packages/expo-router/src/__tests__/loadStaticParamsAsync.test.web.ts

```diff
@@ -497,8 +536,9 @@ describe(loadStaticParamsAsync, () => {
               route: '(index)/blog/123',
             },
             {
+              type: 'route',
               children: [],
-              contextKey: './(app)/(index)/blog/abc.tsx',
+              contextKey: './(app)/(index,about)/blog/abc.tsx',
```

Comment:
contextKey is used as a unique identifier, it looks like it's possible for duplicates now?

Reply:
`contextKey` is being used both as an ID and as a filename for various purposes.

The only place its used as an ID is the `useContextKey()` hook for the navigators. I don't think this breaks their functionality, but we so don't have great test coverage for this :thinking:

I'll add an `id` attribute to `RouteNode` which was the previous `contextKey` value and update `useContextKey` to return the ID value.